new-module/vaults/token-reactor.md

```diff
@@ -0,0 +1,15 @@
+---
+description: Minting agEUR from specific tokens
+---
+
+# 🛩 Token Reactors
+
+Thanks to the previous features of whitelisting addresses and accepting riskier collaterals with tailored parameters, Angle Borrow module could also be used to mint agEUR for specific use cases only. 
+
+## Investing minted agTokens
+
+For example, we could imagine a situation in which the agTokens minted from a Token Reactor could be used only by a whitelisted strategy contract, which would invest them directly on behalf of users. The parameters for this type of vault would be set according to the risk of the assets accepted as collateral, and the collateral ratio of the vault would be programatically managed by the underlying strategy to reduce the chances of getting liquidated. 
```

Comment:
Here it's not a TokenReactor, it is from a specific vault type

Reply:
And let's not use would: 

Angle Borrowing Module has a reactor mode. It allows DAOs to get yield on their idle treasury assets ... and so on